Vagrantfile

```diff
@@ -22,6 +23,12 @@ if options['github_token'].nil? || options['github_token'].to_s.length != 40
   exit
 end
 
+# check host ip
+if options['ip'] == "1"
```

Comment:
why should IP ever be 1? It should be an IP address like here: https://github.com/yiisoft/yii2-app-advanced/blob/master/vagrant/config/vagrant-local.example.yml#L16